src/idiom58.fuzion

```diff
@@ -0,0 +1,2 @@
+file := fuzion.std.fileio
+p := strings.fromBytes(file.read "/home/user/file.xyz")
```

Comment:
When you have to put `(` / `)` around an argument, please separate the argument form the called function by a space. Having no space here means you are using the call syntax using `(arg1, args2, arg3)`, which might be removed.